configure.ac

```diff
@@ -85,6 +85,7 @@ AM_CONDITIONAL([REGEN_BENCHMARKS],
 
 # checks for programs
 AC_PATH_PROG(BISON, bison, no)
+AC_CHECK_PROGS(PYTHON, [python3 python])
```

Comment:
Note that python2 shall not pass!

Reply:
My assumption here is that systems might have python3 under two names in system: `python3` (probably most popular) and `python` (for ones that removed `python2` and don't plan to go back).

Reply:
Right, makes sense.